docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
<p>Again, please update for a Heron processor.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>